lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -83,7 +84,7 @@ public function configureOptions(ConfigureOptionsEvent $event)
      *
      * @param HydrateEvent $event
      */
-    public function handleDefaultLocale(HydrateEvent $event)
+    public function handleDefaultLocale(AbstractMappingEvent $event)
```

Comment:
I can only tell you that this is probably a change breaking Sulu in its current state. As already said Sulu relies on this behavior, since we are saving stuff from time to time which is not locale related (like permissions). Therefore it is critical for me that this behavior does not change.